tests/foreman/cli/test_host.py

```diff
@@ -1316,3 +1316,19 @@ def test_negative_unregister_and_pull_content(self):
         result = self.client.run(
             'yum install -y {0}'.format(FAKE_1_CUSTOM_PACKAGE))
         self.assertNotEqual(result.return_code, 0)
+
+
+class HostErrataTestCase(CLITestCase):
+    """Tests for errata's host sub command"""
+
+    @tier1
+    def test_positive_host_itself_errata_listing(self):
+        """Check if can list errata from the host itself
+
+        @id: 6b22f0c0-9c4b-11e6-ab93-68f72889dc7f
+
+        @assert: Host itself can list errata
+
+        @BZ: 1351040
+        """
+        self.assertIsInstance(Host.errata_list({'host-id': 1}), list)
```

Comment:
Is it certain that the host running the server has always id 1? Should we use the hostname instead, or determine the id from the hostname on the fly? 

Also, I don't think this scenario will work by default, as the host needs to be self-registered to see it's own errata. I didn't try that but downstream docs suggest so: https://access.redhat.com/documentation/en/red-hat-satellite/6.2/paged/server-administration-guide/chapter-3-configuring-a-self-registered-satellite